gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageImpl.java

```diff
@@ -398,6 +399,11 @@ public GoogleCloudStorageImpl(
             ? null
             : this.storage.getRequestFactory().getInitializer();
 
+    if (this.storageOptions.isMetricsEnabled()) {
+      this.metricsPublisher =
+          StackDriverMetricsPublisher.create(options.getProjectId(), credentials);
```

Comment:
Use CloudMonitoring instead of StackDriver, because StackDriver is an obsolete name